ding0/grid/lv_grid/build_grid_on_osm_ways.py

```diff
@@ -39,6 +40,118 @@
 logger = logging.getLogger(__name__)
 
 
+
+def partition_network(network, n_parts=2, node_weight='load', edge_weight='length', contiguous=True):
+    """Partition a network graph into multiple parts using the Metis graph partitioning algorithm.
+
```

Comment:
replace with:
Partition a network graph into multiple parts using the METIS graph partitioning software with
python wrapper PyMetis (k-way graph partitioning is applied, see
G. Karypis, V. Kumar: A fast and high quality multilevel scheme for partitioning irregular graphs
https://www.cs.utexas.edu/~pingali/CS395T/2009fa/papers/metis.pdf)